### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by NistorElena-Simona**

### CODE REVIEW

File: ExamScheduler.Server/Source/Models/User.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace ExamScheduler.Server.Source.Models
+{
+    public class User
+    {
+        [Key]
+        public int user_id { get; set; }
```
Comment (B3N14M1N) [inline ExamScheduler.Server/Source/Models/User.cs:9]: @NistorElena-Simona Coding Style not corresponding

**Review by Alex-Adochitei — COMMENTED** [review]
Frumos Design, dar lipseste conexiunea la baza de date..

File: ExamScheduler.Server/Source/Service/UserService.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        public async Task AddUser(string name, string email, string password, int? groupId = null)
+        {
+            if (string.IsNullOrWhiteSpace(name) || string.IsNullOrWhiteSpace(email) || string.IsNullOrWhiteSpace(password))
+            {
+                throw new ArgumentException("Name, email, and password are required.");
+            }
+
+            // Determinăm rolul pe baza email-ului
+            string role;
+            if (email.Contains("@student"))
```
Comment (B3N14M1N) [inline ExamScheduler.Server/Source/Service/UserService.cs:33]: @NistorElena-Simona Roles shouldn't be hardcoded.

File: examscheduler.client/src/SignUp.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    .signup-container input {
+        width: 100%;
+        padding: 12px;
+        margin-bottom: 15px;
+        border: none;
+        border-radius: 5px;
+        font-size: 1rem;
+    }
+
+    /* Buton */
```
Comment (Alex-Adochitei) [inline examscheduler.client/src/SignUp.css:37]: Butoane prea mici

**Pull request closed without merging**